validator/coordinator/src/main/java/tech/pegasys/teku/validator/coordinator/ValidatorApiHandler.java

```diff
@@ -345,6 +365,24 @@ private ValidatorDuties getDutiesForValidator(
         .orElseGet(() -> ValidatorDuties.noDuties(key));
   }
 
+  private List<ValidatorDuties> getValidatorDutiesFromIndexes(
+      final BeaconState state, final UInt64 epoch, final Collection<Integer> validatorIndexes) {
+    return validatorIndexes.stream()
+        .map(index -> createValidatorDuties(state, epoch, index))
+        .collect(toList());
+  }
+
+  private ValidatorDuties createValidatorDuties(
+      final BeaconState state, final UInt64 epoch, final Integer validatorIndex) {
+    try {
+      BLSPublicKey pkey = state.getValidators().get(validatorIndex).getPubkey();
+      return createValidatorDuties(Map.of(), pkey, state, epoch, validatorIndex);
```

Comment:
This is way into the personal preference category but I tend to use `Collections.emptyMap()` over `Map.of()` as it reads better. Works either way though.